docs/install/scala_setup.md

```diff
@@ -116,6 +117,33 @@ mvn install
 
 <hr>
 
+## Interpreter
+
+To run the scala interpreter, first download and install scala 2.11.x (run `scala -version` to make sure you have the right version installed.**
+
+### Installing the Interpreter
+
+**Ubuntu***
+
+```
+sudo apt-get install scala
```

Comment:
Seems like this could be more specific. Like maybe:
```
wget www.scala-lang.org/files/archive/scala-2.11.7.deb
sudo dpkg -i scala-2.11.7.deb
```
Or whatever minor version is recommended.

Reply:
It should be fine. Scala uses a slightly different versioning scheme (https://docs.scala-lang.org/overviews/core/binary-compatibility-for-library-authors.html) so with version x.y.z, changes to either x or y can be breaking but changes to z are safe. Apt-get installs 2.11.x on both 16.04 and 18.04 so I think it is better to give the simpler command since it is valid.